lvmd/command/lvm.go

```diff
@@ -220,58 +163,40 @@ func (g *VolumeGroup) FindVolume(name string) (*LogicalVolume, error) {
 
 // ListVolumes lists all logical volumes in this volume group.
 func (g *VolumeGroup) ListVolumes() ([]*LogicalVolume, error) {
```

Comment:
This function no longer returns an error.

Reply:
Done, also changed other list methods which no longer return `error` as well.